linode/manager
# feat: [M3-6740] - Populate Subnets table with data

## Description 📝
Add VPC Subnets Table to the VPC details page. Action menu is stubbed for now

## Preview 📷
https://github.com/linode/manager/assets/115299789/446ab930-a3c5-44c0-82e7-744b7cb35b2b

Empty State

![subnets empty state](https://github.com/linode/manager/assets/115299789/5b9b59da-a5d7-4179-8ced-b42855f8a118) 
![subnet linode empty state](https://github.com/linode/manager/assets/115299789/a0b6fcd6-9596-4422-adb3-ff580eb85d86)

## How to test 🧪
```
yarn test VPCSubnetsTable
```
```
yarn test SubnetLinodeRow
```
Turn on the MSW, go to a VPC's details page and check the Subnets table
